frontend/dfx_react/src/dfx_react/public/index.jsx

```diff
@@ -0,0 +1,42 @@
+import "regenerator-runtime/runtime.js";
+
+import reactCanister from 'ic:canisters/dfx_react';
+
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom';
+
+async function asyncUpdateCount(val, callback) {
```

Comment:
I noticed this in both examples. Is there a reason to use `val` as a string and use a switch/case?

See https://github.com/dfinity/examples/pull/30/files#r433553269.